cypress/e2e/cloud/releases.test.ts

```diff
@@ -0,0 +1,146 @@
+      beforeEach(() => {
+        cy.signinWithoutUserReprovision()
+      })
+
+      it('depreciated features do not exist', () => {
+        cy.getByTestID('nav-item-flows').should('not.exist')
+        cy.getByTestID('nav-item-alerting').should('not.exist')
+        cy.getByTestID('nav-item-dashboards').should('not.exist')
+        cy.getByTestID('nav-item-tasks').should('not.exist')
+      })
+
+      describe('supported features are still navigable', () => {
+        ;[
+          {navId: 'nav-item-data-explorer', title: 'Data Explorer'},
+          {navId: 'nav-item-load-data', title: 'Load Data'},
+          {navId: 'nav-item-settings', title: 'Settings'},
+        ].forEach(({navId, title}) => {
+          it(`for ${title} page`, () => {
+            cy.getByTestID(navId).should('be.visible').click({force: true})
+            cy.getByTestID('page-title').contains(title)
+          })
+        })
+      })
+
+      describe('nav subItems are navigable based upon selective support', () => {
+        ;[
+          {
+            navId: 'nav-item-load-data',
+            navSubId: 'nav-subitem-sources',
+            tabId: 'sources--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-load-data',
+            navSubId: 'nav-subitem-buckets',
+            tabId: 'buckets--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-load-data',
+            navSubId: 'nav-subitem-telegrafs',
+            tabId: 'telegrafs--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-load-data',
+            navSubId: 'nav-subitem-tokens',
+            tabId: 'tokens--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-load-data',
+            navSubId: 'nav-subitem-subscriptions',
+            tabId: 'subscriptions--tab',
+            supported: false,
+          },
+          {
+            navId: 'nav-item-settings',
+            navSubId: 'nav-subitem-labels',
+            tabId: 'labels--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-settings',
+            navSubId: 'nav-subitem-secrets',
+            tabId: 'secrets--tab',
+            supported: true,
+          },
+          {
+            navId: 'nav-item-settings',
+            navSubId: 'nav-subitem-templates',
+            tabId: 'templates--tab',
+            supported: false,
+          },
+        ].forEach(({navId, navSubId, tabId, supported}) => {
+          it(`${navSubId} is ${
+            supported ? 'supported' : 'not supported'
+          }`, () => {
+            cy.getByTestID(navId).should('be.visible').click({force: true})
+            cy.getByTestID(navSubId).should(supported ? 'exist' : 'not.exist')
+            cy.getByTestID(tabId).should(supported ? 'be.visible' : 'not.exist')
+          })
+        })
+      })
+    })
+
+    // TODO: add any more marty-specific features
```

Comment:
Do we have any marty-specific features, which do not exist for the other/older iox orgs?